recipes/libmodplug/all/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_definitions("-DHAVE_STDINT_H")
```

Comment:
this should be portable enough, I support (nowadays we can assume every compiler has **stdint.h**, I hope).
otherwise, we may add something like [CheckIncludeFile](https://cmake.org/cmake/help/v3.14/module/CheckIncludeFile.html)
not a blocker, just a random note, maybe for the future portability